mkdocs.yml

```diff
@@ -239,6 +239,7 @@ nav:
               - JS SDK Official Doc: https://docs.bnbchain.org/greenfield-js-sdk/
           - How to Contribute: ./bnb-greenfield/for-developers/contribute.md
 
+          - Blob Hub: ./bnb-greenfield/for-developers/blob-hub.md
```

Comment:
I think the order is also important, `How to Contribute` is always the last one,  `bundle service` and `blob hub` is close related, they should be the bottom ones.

Reply:
moved